PowerPointLabs/PowerPointLabs/PictureSlidesLab/ViewModel/PictureSlidesLabWindowViewModel.VariationStageControls.cs

```diff
@@ -88,7 +90,82 @@ public void BindSelectedFont(Slide contentSlide, float slideWidth, float slideHe
                 View.CreateDefaultPictureItem(),
                 contentSlide,
                 slideWidth,
-                slideHeight);
+                slideHeight,
+                true);
+        }
+        #endregion
+
+        #region Binding funcs for slider
+        public void BindStyleToSlider()
+        {
+            if (!IsAbleToBindProperty()) return;
+
+            var styleOption = _styleOptions[StylesVariationListSelectedId.Number];
+            var currentCategory = CurrentVariantCategory.Text;
+            var type = styleOption.GetType();
+
+            if (currentCategory == TextCollection.PictureSlidesLabText.VariantCategoryBlurriness)
```

Comment:
so many `if (category == ...) then do ...` here and there, they can be refactored into like factory pattern:
if it's certain category, factory (SliderPropHandlerFactory) get the proper handler (e.g. BlurrinessSliderPropHandler) to handle certain cases (e.g. setupSliderProperties, bindStyleOption, bindStyleVariant). without this pattern, this chain of conditions will become longer and hard to maintain in the future